PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+**Note:** Filling out this template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the
```

Comment:
We'd discussed conveying our expectations of iterative development, meaning that changes are well-defined and of a manageable size. Do you think these expectations are already covered? If not, would they be best added now or in a subsequent iteration?